fluid/DeepASR/data_utils/load_data.py

```diff
@@ -0,0 +1,183 @@
+#by zhxfl 2018.01.24
+""" @package docstring
+load speech data from disk
+"""
+
+import random
+import Queue
+import numpy
+import struct
+import data_utils.trans_mean_variance_norm as trans_mean_variance_norm
+import data_utils.trans_add_delta as trans_add_delta
+
+g_lblock = []
+g_que_sample = Queue.Queue()
+g_nframe_dim = 120 * 11
+g_nstart_block_idx = 0
+g_nload_block_num = 1
+g_ndrop_frame_len = 256
+
+class OneBlock(object):
+    """ Documentation for a class.
+        struct for one block :
+        contain label, label desc, feature, feature_desc
+    """
+    def __init__(self):
+        """The constructor."""
+        self.label = ""
+        self.label_desc = ""
+        self.feature = ""
+        self.feature_desc = ""
+
+def set_trans(ltrans):
+    global g_ltrans
+    g_ltrans = ltrans
+
+def load_list(sfeature_lst, slabel_lst):
+    """ load list """
+    global g_lblock 
+
+    lFeature = open(sfeature_lst).readlines()
+    lLabel = open(slabel_lst).readlines()
+    assert len(lLabel) == len(lFeature)
+    for i in range(0, len(lFeature), 2):
+        one_block = OneBlock()
+
+        one_block.label = lLabel[i]
+        one_block.label_desc = lLabel[i + 1]
+        one_block.feature = lFeature[i]
+        one_block.feature_desc = lFeature[i + 1]
+        g_lblock.append(one_block)
+    
+    random.shuffle(g_lblock)
+
+def load_one_block(lsample, id):
+    """read one block"""
```

Comment:
Please give more detailed comments and you can refer http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html